demosplan/DemosPlanCoreBundle/Logic/Statement/AssessmentTableExporter/AssessmentTableZipExporter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the package demosplan.
+ *
+ * (c) 2010-present DEMOS plan GmbH, for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Statement\AssessmentTableExporter;
+
+use demosplan\DemosPlanCoreBundle\Entity\File;
+use Exception;
+use PhpOffice\PhpSpreadsheet\Writer\Xlsx;
+
+class AssessmentTableZipExporter extends AssessmentTableXlsExporter
+{
+    protected array $supportedTypes = ['zip'];
+
+    /**
+     * @throws Exception
+     */
+    public function __invoke(array $parameters): array
+    {
+        $xlsxArray = parent::__invoke($parameters);
+
+        $statementAttachments = $this->getAttachmentsOfStatements($parameters['items']);
+
+        /** @var Xlsx $xlsxWriter */
+        $xlsxWriter = $xlsxArray['writer'];
```

Comment:
is key writer always defined and a Xlsx?

Reply:
I think so, because in `$this->spreadsheetFactory->createWriter($document, 'Xlsx');` otherwise an error would have to be thrown.